sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -22,7 +22,7 @@
 """Create, read, update and delete items in the Azure Cosmos DB SQL API service.
```

Comment:
Were you going to rename these files in this PR? Or separately? (i.e. -> _container.py). I don't mind if we do it separately if we want to get this PR in.